source/stylesheets/refills/unstyled/_hover-menu.scss

```diff
@@ -0,0 +1,68 @@
+.hover-menu {
+  $large-screen: em(860) !default;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets